examples/04-Exporting-ranking-models.ipynb

```diff
@@ -0,0 +1,380 @@
+{
```

Comment:
<p>Can we a description of the used function for NVTabular?</p><p> </p><p>after the workflow is saved, can we show the output with !tree folder?</p><p> </p><p><code></code><code>python</p><p>workflow = nvt.Workflow(outputs)</p><p>...</p><p>workflow.save(path)</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
not sure I understand the comment: `Can we a description of the used function for NVTabular?`